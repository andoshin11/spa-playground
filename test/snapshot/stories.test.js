import path from 'path'
import initStoryshots from '@storybook/addon-storyshots'

initStoryshots({
  framework: 'vue',
  configPath: path.join(__dirname, '../../.storybook')
})
