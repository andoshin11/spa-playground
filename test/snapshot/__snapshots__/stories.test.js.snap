// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots components/Base/Button Default 1`] = `
<div
  class="Storybook"
>
  <div
    class="StorybookSection"
  >
    <div
      class="StorybookSection__Header"
    >
      Primary Buttons
    </div>
     
    <div
      class="StorybookSection__Body"
    >
      <div
        class="StorybookSection__Label"
      >
        Large
      </div>
       
      <button
        class="Button -primary -large"
        type="button"
      >
        <div
          class="Button__Text"
        >
          test
        </div>
         
        <!---->
      </button>
       
      <div
        class="StorybookSection__Label"
      >
        Midium
      </div>
       
      <button
        class="Button -primary -midium"
        type="button"
      >
        <div
          class="Button__Text"
        >
          test
        </div>
         
        <!---->
      </button>
       
      <div
        class="StorybookSection__Label"
      >
        Small
      </div>
       
      <button
        class="Button -primary -small"
        type="button"
      >
        <div
          class="Button__Text"
        >
          test
        </div>
         
        <!---->
      </button>
    </div>
  </div>
</div>
`;

exports[`Storyshots components/Base/Button Disabled 1`] = `
<div
  class="Storybook"
>
  <div
    class="StorybookSection"
  >
    <div
      class="StorybookSection__Header"
    >
      Primary Buttons
    </div>
     
    <div
      class="StorybookSection__Body"
    >
      <div
        class="StorybookSection__Label"
      >
        Large
      </div>
       
      <button
        class="Button -primary -large"
        disabled="disabled"
        type="button"
      >
        <div
          class="Button__Text"
        >
          test
        </div>
         
        <!---->
      </button>
       
      <div
        class="StorybookSection__Label"
      >
        Midium
      </div>
       
      <button
        class="Button -primary -midium"
        disabled="disabled"
        type="button"
      >
        <div
          class="Button__Text"
        >
          test
        </div>
         
        <!---->
      </button>
       
      <div
        class="StorybookSection__Label"
      >
        Small
      </div>
       
      <button
        class="Button -primary -small"
        disabled="disabled"
        type="button"
      >
        <div
          class="Button__Text"
        >
          test
        </div>
         
        <!---->
      </button>
    </div>
  </div>
</div>
`;

exports[`Storyshots components/Base/Pager Default 1`] = `
<div
  class="Storybook"
>
  <div
    class="StorybookSection"
  >
    <div
      class="StorybookSection__Header"
    >
      Pager
    </div>
     
    <div
      class="StorybookSection__Body dark"
    >
      <div
        class="wrapper"
        style="width: 100%;"
      >
        <div
          style="color: rgb(255, 255, 255); margin-bottom: 16px; font-size: 18px; font-weight: bold;"
        >
          Current Page: 3
        </div>
         
        <div
          class="Pager"
        >
          <button
            class="Pager__ArrowButton"
          >
            ←
          </button>
           
          <button
            class="Pager__ArrowButton"
          >
            →
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots components/Base/Pager Last page 1`] = `
<div
  class="Storybook"
>
  <div
    class="StorybookSection"
  >
    <div
      class="StorybookSection__Header"
    >
      Pager
    </div>
     
    <div
      class="StorybookSection__Body dark"
    >
      <div
        class="wrapper"
        style="width: 100%;"
      >
        <div
          style="color: rgb(255, 255, 255); margin-bottom: 16px; font-size: 18px; font-weight: bold;"
        >
          Current Page: 10
        </div>
         
        <div
          class="Pager"
        >
          <button
            class="Pager__ArrowButton"
          >
            ←
          </button>
           
          <button
            class="Pager__ArrowButton"
            disabled="disabled"
          >
            →
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots components/Modules/Movies/Poster Default 1`] = `
<div
  class="Storybook"
>
  <div
    class="StorybookSection"
  >
    <div
      class="StorybookSection__Header"
    >
      Poster
    </div>
     
    <div
      class="StorybookSection__Body"
    >
      <div
        class="wrapper"
        style="width: 320px;"
      >
        <div
          class="Poster"
          style="height: 0px;"
        >
          <img
            alt="Ant-Man and the Wasp"
            class="Poster__Image"
            src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots components/Modules/Movies/PosterList Default 1`] = `
<div
  class="Storybook"
>
  <div
    class="StorybookSection"
  >
    <div
      class="StorybookSection__Header"
    >
      Poster
    </div>
     
    <div
      class="StorybookSection__Body dark"
    >
      <div
        class="wrapper"
        style="width: 100%;"
      >
        <div
          class="PosterList"
        >
          <div
            class="PosterList__Header"
          >
            <div
              class="PosterList__Title"
            >
              Popular Movies
            </div>
             
            <div
              class="PosterList__Nav"
            >
              <div
                class="Pager"
              >
                <button
                  class="Pager__ArrowButton"
                  disabled="disabled"
                >
                  ←
                </button>
                 
                <button
                  class="Pager__ArrowButton"
                >
                  →
                </button>
              </div>
            </div>
          </div>
           
          <div
            class="PosterList__Body"
          >
            <div
              class="PosterList__Slider"
            >
              <div
                class="PosterList__SliderInner"
                style="left: 5vw;"
              >
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
                <div
                  class="PosterList__SliderItem"
                >
                  <div
                    class="Poster"
                    style="height: 0px;"
                  >
                    <img
                      alt="Ant-Man and the Wasp"
                      class="Poster__Image"
                      src="https://image.tmdb.org/t/p/w600_and_h900_bestv2//rv1AWImgx386ULjcf62VYaW8zSt.jpg"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
